extensions/2.0/Khronos/KHR_materials_volume/README.md

```diff
@@ -0,0 +1,253 @@
+
+where T is commonly referred to as *transmittance*. 
+
+Substituting σ<sub>t</sub> in the previous equation by its definition via *attenuation color* and *attenuation distance*, as defined above, and setting *x* = *d* we get
+
+T(d<sub>a</sub>) = e<sup>(-log(*c*) / *d*) * *d*</sup> = *c*
+
+So, after traveling distance *d* through the medium we get attenuation color *c*.
+
+## Base Color and Absorption
+
+Base color and absorption both have an effect on the final color of a volumetric object, but the behavior is different. Base color changes the color of light at the volume boundary. Absorption occurs while the light is traveling through the volume. Depending on the distance the light travels, more or less of it is absorbed, making the overall color of the object dependent on its shape.
+
+<figure style="text-align:center">
+<img src="./figures/base-color-absorption.png"/>
+<figcaption><em>Base color changes the amount of light passing through the volume boundary (left). The overall color of the object is the same everywhere, as if the object is covered with a colored, transparent foil. Absorption changes the amount of light traveling through the volume (right). The overall color depends on the distance the light traveled through it; at small distances (tail of the dragon) less light is absorbed and the color is brighter than at large distances.</em></figcaption>
+</figure>
+
+## Transmission and Translucency
+
+The volume extension needs one of `KHR_materials_transmission` and `KHR_materials_translucency` to allow light rays to pass through the surface into the volume. Once the volume is entered however, the simulation of absorption and potentially subsurface scattering inside the medium is independent of the surface properties.
+
+If the extension is combined with `KHR_materials_transmission`, a refractive microfacet BTDF describes the transmission of light through the volume boundary. The refraction occurs on microfacets, and thus the roughness parameter affects both reflection and transmission. An exemplatory implementation is shown in [Implementation](#Implementation).
+
+<figure style="text-align:center">
+<img src="./figures/transmissive-roughness.png"/>
+<figcaption><em>Transmissive sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption>
+</figure>
+
+If the extension is combined with `KHR_materials_translucency`, a diffuse transmission BTDF describes the transmission of light through the volume boundary. The object becomes translucent. The roughness parameter only affects the reflection. `KHR_materials_translucency` may be a good choice if the appearance should match a material which exhibits strong subsurface scattering. Examples for these dense materials are skin or candle wax. 
+
+<figure style="text-align:center">
+<img src="./figures/translucent-roughness.png"/>
+<figcaption><em>Translucent sphere with varying roughness. From left to right: 0.0, 0.2, 0.4.</em></figcaption>
+</figure>
```

Comment:
This section needs to move to one of the other draft extensions, such as SSS or translucency. (If the latter, it should get updated screenshots showing thin-walled differences between transmission and translucency).

Reply:
I'm on it